warplib/src/warp_memory.cairo

```diff
@@ -1,5 +1,16 @@
 use dict::Felt252Dict;
 use dict::Felt252DictTrait;
+use integer::u256_from_felt252;
+
+mod arrays;
+use arrays::WarpMemoryArraysTrait;
+
+mod bytes;
+use bytes::WarpMemoryBytesTrait;
+
+mod accessors;
+use accessors::WarpMemoryAccesssorTrait;
+use accessors::WarpMemoryMultiCellAccessorTrait;
 
 
 struct WarpMemory {
```

Comment:
Since it's the more or less final version of warp memory we should come up with a better name than `pointer` imo. It's meaningless. Do you have any preferred name? Maybe we should just name it `length`? Or at least `free_space_pointer`?

Reply:
I like `free_space_pointer`